agent/conversion_service.py

```
@@ -0,0 +1,312 @@
+        # Wait for confirmation
+        receipt = self.w3.eth.wait_for_transaction_receipt(tx_hash, timeout=300)
+        
+        if receipt.status != 1:
+            raise Exception(f"Swap transaction failed: {tx_hash.hex()}")
+        
+        return tx_hash.hex(), receipt
+    
+    def convert_usdt0_to_usdc(
+        self,
+        usdt0_amount: int,
+        user_address: str,
```

Ruff (0.14.4)

Unused method argument: `user_address`

(ARG002)